command/launch.go

```diff
@@ -64,6 +69,11 @@ func (c *LaunchCommand) Run(args []string) int {
 		flags.Parse(flags.Args()[1:])
 	}
 
+	if arn != "" && roleName != "" {
+		fmt.Fprintln(os.Stderr, "InvalidParameterCombination: The parameter 'roleName' and 'arn' can not be used in combination")
```

Comment:
Is this message copied from original error of aws-sdk-go?
If so, please show it as it is.

```go
fmt.Fprintln(os.Stderr, err)
return 1
```

(Of course, I have to treat all other errors as well as this...)